vignettes/sidebars.Rmd

```diff
@@ -58,215 +87,398 @@ iframe, .bslib-page-fill {
+
+:::: {.row}
+::: {.col-sm-12 .col-lg-6}
+### Shared Sidebar {#intro-shared-sidebar}
+
+When `sidebar` is an argument to a function containing multiple pages or panels, it creates a **[shared sidebar](#shared-sidebar)**.
+
+Provide a `sidebar()` to `page_navbar()` to recreate the classic [shinydashboard](http://rstudio.github.io/shinydashboard/) look with a global sidebar [shared across several pages](#navbar-pages).
+
+You can also add a [shared sidebar to a card with tabs or pills](#card-tabs-shared-sidebar) created with `navs_tab_card()` or `navs_pill_card()`.
+:::
+
+::: {.col-sm-12 .col-lg-6 .text-center .mt-2 .mt-lg-4}
+```{r preview-page-navbar, echo = FALSE}
+describe_layout_function(
+  "page_navbar()",
+  "A shared sidebar in a multi-page layout.",
+  card_body_fillable(
+    class = "p-0",
+    height = 200,
+    id = "page_navbar_example_card",
+    navs_bar(
+      collapsible = FALSE,
+      fillable = TRUE,
+      sidebar = sidebar("Shared sidebar", width = "33%", class = "bg-light"),
+      nav("Page 1", "Page 1 content"),
+      nav("Page 2", "Page 2 content")
+    ),
+    tags$script(HTML("$(document).ready(function() {
+      var demo_nav = $('#page_navbar_example_card nav.navbar');
+      setTimeout(function(){ demo_nav.headroom('destroy') }, 100);
+    });"))
+  ) |>
+  tagAppendAttributes(class = "flex-row", .cssSelector = ".navbar-nav") |>
+  tagAppendAttributes(class = "d-none", .cssSelector = ".navbar-header") |>
+  tagAppendAttributes(class = "navbar-dark", .cssSelector = "nav")
+)
+```
 
-To create a sidebar layout, provide a `sidebar()` to the `sidebar` argument of one of the following functions: 
+```{r preview-navs-tab-card, echo = FALSE}
+describe_layout_function(
+  c("navs_tab_card()", "navs_pill_card()"),
+  "A shared sidebar for tabs within a card.",
+  card_body_fillable(
+    class = "p-2",
+    height = 200,
+    navs_tab_card(
+      title = "Tab Card",
+      sidebar = sidebar("Shared sidebar", width = "33%", class = "bg-light"),
+      nav("Tab 1", "Tab 1 content"),
+      nav("Tab 2", "Tab 2 content")
+    )
+  )
+)
+```
+:::
+::::
+
+:::: {.row}
+::: {.col-sm-12 .col-lg-6}
+### Local Sidebar {#intro-local-sidebar}
+
+Use `layout_sidebar()` whenever you want a **[local sidebar](#local-sidebar)** adjacent to a content panel.
+`layout_sidebar()` creates a bordered area by default, making it useful anywhere in your interface.
+
+It also fits nicely into [card components](#cards) and [nav pages or tabs](#navbar-pages) allowing you to add a sidebar to individual pages or tabs.
+:::
+
+::: {.col-sm-12 .col-lg-6 .text-center .mt-2 .mt-lg-4}
+```{r preview-layout-sidebar, echo = FALSE}
+describe_layout_function(
+  "layout_sidebar()",
+  "A flexible, local sidebar layout.",
+  card_body_fillable(
+    class = "p-2",
+    height = 200,
+    layout_sidebar(
+      sidebar = sidebar("Local sidebar", width = "33%", class = "bg-light"),
+      "Content with a sidebar"
+    )
```

Comment:
The border/border-radius is removed/changes since it's within a card, but you can add them back these utility classes

```suggestion
    layout_sidebar(class = "border rounded-top",
      sidebar = sidebar("Local sidebar", width = "33%", class = "bg-light"),
      "Content with a sidebar"
    )
```